src/front/feedback/FeedbackContainer.tsx

```diff
@@ -119,19 +122,26 @@ extends React.Component <any, FeedbackContainerState> {
 		})
 		.then((response) => {
 			console.dir(response);
-
 			this.setState({
 				isFeedbackReady: true,
+				isSending: false,
 				articleItems: items,
 				updateToken: response.updateToken,
 			});
 		});
 	}
 
 	public render() {
-		return this.state.isFeedbackReady
-			? this.renderConfirmationPage()
-			: this.renderFeedbackForm();
+		const { isFeedbackReady, isSending } = this.state;
+		const feedbackForm = this.renderFeedbackForm();
+		const confirmationPage = this.renderConfirmationPage();
+		return (
+			<div>
+			{ isFeedbackReady && !isSending && confirmationPage }
+			{ !isFeedbackReady && !isSending && feedbackForm }
+			{ isSending && !isFeedbackReady &&  <Spinner/> }
+			</div>
+		);
```

Comment:
Probably better use instant function not variable.
And maybe better optimize conditions, if you are sending something, you don't need look at feedback variable, and something similar happened when feedback was returned )

{ !isFeedbackReady && !isSending &&  this.renderFeedbackForm(); }
{ isFeedbackReady && this.renderConfirmationPage(); }
{ isSending && <Spinner/> }